Unaccountable-Magpie/WorkforceManagement
# BS Departments Controller

### Purpose:

Allows HR to create new departments.


### How it fits in context of the project:


Central functionality of all things depending on departments.


### Specific Feature affected:


Departments Controller


### How to test (Be thorough!):

1. Pull down and check into my branch bs-departments

2. Load the DepartmentsController.cs file and Run it

3. Check to see there are no build errors

4. Further testing will require the implementation of the currently unmerged Departments View
